shared-src/net/wurstclient/features/mods/other/MurderMysteryMod.java

```diff
@@ -0,0 +1,178 @@
+import net.wurstclient.events.listeners.UpdateListener;
+import net.wurstclient.features.Mod;
+import net.wurstclient.features.SearchTags;
+import net.wurstclient.utils.ChatUtils;
+import net.wurstclient.utils.RenderUtils;
+
+@SearchTags({"hypixel"})
+@Mod.Bypasses
+public final class MurderMysteryMod extends Mod
+	implements UpdateListener, RenderListener
+{
+	private static final AxisAlignedBB PLAYER_BOX =
+		new AxisAlignedBB(-0.35, 0, -0.35, 0.35, 1.9, 0.35);
+	private String murderer = "";
+	private boolean renderNotice = false;
+
+	public MurderMysteryMod()
+	{
+		super("MurderMystery",
+			"Tells you who is the murderer in Murder Mystery."
+				+ "\nJust keep the mod enabled, the murderer will updated once the new murderer is found.");
+		// setCategory(Category.OTHER);
+	}
+
+	@Override
+	public String getRenderName()
+	{
+		if(murderer.equals(""))
+			return "MurderMystery [Scanning]";
+		else
+			return "MurderMystery [Murderer: " + murderer + "]";
+	}
+
+	@Override
+	public void onDisable()
+	{
+		wurst.events.remove(UpdateListener.class, this);
+		wurst.events.remove(RenderListener.class, this);
+	}
+
+	@Override
+	public void onEnable()
+	{
+		murderer = "";
+		renderNotice = false;
+		wurst.events.add(UpdateListener.class, this);
+		wurst.events.add(RenderListener.class, this);
+		if(isUserNotMurderer())
+			whoHasSword();
+	}
+
+	@Override
+	public void onRender(float partialTicks)
+	{
+		if(murderer.equals("") || murderer.equals("YOU!"))
+			return;
+
+		// GL settings
+		GL11.glEnable(GL11.GL_BLEND);
+		GL11.glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
+		GL11.glEnable(GL11.GL_LINE_SMOOTH);
+		GL11.glLineWidth(6);
+		GL11.glDisable(GL11.GL_TEXTURE_2D);
+		GL11.glDisable(GL11.GL_DEPTH_TEST);
+
+		GL11.glPushMatrix();
+		GL11.glTranslated(-mc.getRenderManager().renderPosX,
+			-mc.getRenderManager().renderPosY,
+			-mc.getRenderManager().renderPosZ);
+
+		// draw boxes
+		for(EntityPlayer entity : WMinecraft.getWorld().playerEntities)
+		{
+			if(!entity.getName().equals(murderer))
+				continue;
+
+			// set color
+			GL11.glColor4f(1F, 0, 1, 0.2F);
+
+			// set position
+			GL11.glPushMatrix();
+			GL11.glTranslated(
+				entity.prevPosX
+					+ (entity.posX - entity.prevPosX) * partialTicks,
+				entity.prevPosY
+					+ (entity.posY - entity.prevPosY) * partialTicks,
+				entity.prevPosZ
+					+ (entity.posZ - entity.prevPosZ) * partialTicks);
+
+			// draw box
+			RenderUtils.drawOutlinedBox(PLAYER_BOX);
+			GL11.glPopMatrix();
+		}
+
+		GL11.glPopMatrix();
+
+		// GL resets
+		GL11.glEnable(GL11.GL_DEPTH_TEST);
+		GL11.glEnable(GL11.GL_TEXTURE_2D);
+		GL11.glDisable(GL11.GL_BLEND);
+		GL11.glDisable(GL11.GL_LINE_SMOOTH);
+
+	}
+
+	@Override
+	public void onUpdate()
+	{
+		updateMS();
+		if(!hasTimePassedM(250))
+			return;
+		if(isUserNotMurderer())
+			whoHasSword();
+		updateLastMS();
+	}
+
+	private boolean isUserNotMurderer()
+	{
+		if(murderer.equals("YOU!"))
```

Comment:
@DesWurstes I found a solution for your problem: https://github.com/WurstAPI/Plugin-Examples/blob/master/MurderMystery/src/main/java/me/deathsgun/murdermystery/MurderMysteryMod.java#L167-L183